File: 2137.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        while (sc.hasNextInt()) {
            int N = sc.nextInt();
            String[] books = new String[N];

            for (int i = 0; i < N; i++) {
                books[i] = sc.next();
            }

            Arrays.sort(books);

            for (String b : books) {
                System.out.println(b);
            }
        }

        sc.close();
    }
}

File: 2137.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main() {
    int N;
    while (cin >> N) {
        vector<string> books(N);

        for (int i = 0; i < N; i++) {
            cin >> books[i];
        }

        sort(books.begin(), books.end());

        for (auto &b : books) {
            cout << b << "\n";
        }
    }
    return 0;
}
